frontend/source/components/holed-layer/holed-layer.vue

```diff
@@ -44,6 +44,8 @@ const boxTop = computed(() => {
 	return `${calculatedTop}px`;
 });
 const boxLeft = computed(() => calculateTutorialBoxLeft(currentStep.value.width, currentStep.value.selector, currentElementBoundingRect.value));
+const boxLeftWithPx = computed(() => `${boxLeft.value}px`);
```

Comment:
I can't because I use the pure boxLeft.value just in line 48.
`const arrowLeft = computed(() => `${currentElementBoundingRect.value.left - boxLeft.value}px`);`